common-features/cdn.md

```diff
@@ -0,0 +1,84 @@
+---
+title: CDN
+page_title: Telerik UI for Blazor CDN
+description: Learn how to use the Telerik Blazor CDN to load CSS and JavaScript files.
+slug: common-features-cdn
+tags: telerik,blazor,cdn
+published: True
+position: 100
+---
+
+# Telerik UI for Blazor CDN
```

Comment:
Personally, I would take a different approach to this page and structure it somewhat differently:

1. Intro that is goal/task-oriented, rather than a description of a complimentary service.
2. Next, I would discuss the pros and cons of using the service.
3. Then I would have a section about how to implement the CDN service in my app. I wouldn't really split it into two separate sections for the CSS and the JS files, I would probably list it in an unordered procedure and be brief about it.
4. Finally, I would discuss the fallback implementation. I might even include its basics (no code, but a high-level approach) and link to the respective KB.

Reply:
Pros and cons are general knowledge, that's why they come later in the article. We don't want to make the article sound goal-oriented, because we don't want customers to actually use the CDN :)